client/apollo/react/src/CardMessage/CardMessageCommon.tsx

```diff
@@ -0,0 +1,41 @@
+import { ComponentPropsWithoutRef, useMemo } from "react";
+import { getComponentClassName } from "../utilities/getComponentClassName";
+
+export const cardMessageVariants = {
+  info: "info",
+  warning: "warning",
+  error: "error",
+  neutral: "neutral",
+} as const;
+export type CardMessageVariants = keyof typeof cardMessageVariants;
+
+export type CardMessageProps = ComponentPropsWithoutRef<"div"> & {
+  variant?: CardMessageVariants;
+  title?: string;
+  text: string;
+};
+
+export const CardMessage = ({
+  variant = "info",
+  title,
+  text,
+  className,
+  ...props
+}: CardMessageProps) => {
+  const componentClassName = useMemo(() => {
+    const newClassModifier = [variant].filter(Boolean).join(" ");

```

Comment:
a priori, on ne peut avoir qu'un seul variant donc pas de tableau.
```suggestion
    const newClassModifier = [variant].filter(Boolean).join(" "); // pas utile
getComponentClassName(
      "af-card-message",
      className,
      variant,
    );
```